Dart/src/com/jetbrains/lang/dart/ide/completion/DartServerCompletionContributor.java

```diff
@@ -541,16 +545,26 @@ else if (shouldSetSelection) {
 
 
   @NotNull
-  private static CompletionSuggestion createCompletionSuggestionFromAvailableSuggestion(@NotNull final AvailableSuggestion suggestion,
-                                                                                        @NotNull final AvailableSuggestionSet suggestionSet,
-                                                                                        final int relevance) {
+  private static CompletionSuggestion createCompletionSuggestionFromAvailableSuggestion(
+      @NotNull final AvailableSuggestion suggestion,
+      @NotNull final AvailableSuggestionSet suggestionSet,
+      final int suggestionSetRelevance,
+      Map<String, IncludedSuggestionRelevanceTag> includedSuggestionRelevanceTags) {
+    int relevance = suggestionSetRelevance;
+    for (String tag : suggestion.getRelevanceTags()) {
```

Comment:
I got NPEs here both when running DartServerCompletionTest or when testing manually. Probably 'relevaneTags' field is optional in the protocol, so it appears to be null.
I surrounded this `for` loop with `if (relevanceTags != null) {...}`.